src/window/glfw.rs

```diff
@@ -273,7 +273,7 @@ pub fn initialize_settings() {
         .contains(&String::from("--noIdle"));
 
     SETTINGS.set(&WindowSettings {
-        refresh_rate: 60,
+        refresh_rate: 120,
```

Comment:
Do you have a reason for changing this? Not trying to micromanage, just curious.

Reply:
Yes, you find the root point, I realized that glfw seems performance lower than sdl2, so I just increase that to 120, then seems performance improved. And I am not sure if it has any side effect, stay tuned.

Reply:
Interesting. I will dig into this more once a full PR is ready. I suspect there  are minor differences in the way each windowing system queues a new frame.